packages/js/components/src/date-time-picker-control/date-time-picker-control.tsx

```diff
@@ -265,7 +265,11 @@ export const DateTimePickerControl = forwardRef(
 		}, [ onBlur ] );
 
 		const callOnBlurIfDropdownIsNotOpening = useCallback( ( willOpen ) => {
-			if ( ! willOpen && typeof onBlurRef.current === 'function' ) {
+			if (
+				! willOpen &&
+				typeof onBlurRef.current === 'function' &&
+				inputControl.current
+			) {
```

Comment:
I added the `inputControl.current` check, as I somehow triggered a scenario where `inputControl.current` was null. Although I couldn't reproduce this :/

Reply:
I couldn't reproduce the `inputControl.current` = `null`, but it's good to be defensive here 🙂